src/lib/Components/ArtworkFilter/__tests__/FilterModal-tests.tsx

```diff
@@ -367,7 +367,7 @@ describe("Clearing filters", () => {
     expect(extractText(filterScreen.root.findAllByType(CurrentOption)[0])).toEqual("")
   })
 
-  it("enables the apply button when clearing all if no other options are selected", () => {
+  it("exit modal when clear all button was pressed", () => {
```

Comment:
very minor suggestion to improve readability:

```suggestion
  it("exits the modal when clear all button is pressed", () => {
```

Reply:
Fixed. My bad, sorry

Reply:
Nothing to apologize for! Thanks for the update 👍🏼